handlers/feedback.go

```diff
@@ -140,69 +95,112 @@ func addFeedback(w http.ResponseWriter, req *http.Request, isPositive bool, rend
 	decoder := schema.NewDecoder()
 	decoder.IgnoreUnknownKeys(true)
 
-	var f Feedback
-	if err := decoder.Decode(&f, req.Form); err != nil {
+	var ff model.FeedbackForm
+	if err := decoder.Decode(&ff, req.Form); err != nil {
 		log.Error(ctx, "unable to decode request form", err)
 		w.WriteHeader(http.StatusInternalServerError)
```

Comment:
Appreciate this isn't part of your change, but do you think this would be better if it sent a 400 response?

Reply:
hmm this is a bit more tricky; if we go with the user has changed some of the names of the fields then a 400 is a reasonable response but if the decoding fails for another reason is that a client error?